constants/applicationWorth.js

```diff
@@ -31,6 +31,8 @@ const applicationWorth = {
   prestigeItem: 1,
   reforge: 1,
   winningBid: 1,
+  polarvoid: 1,
+  divanPowderCoating: 1,
```

Comment:
I try to keep these grouped by value and then sorted alphabetical within each group.

Reply:
Not entirely sure how you group it, I tried to guess. If it's not correct just create a commit suggestion